__tests__/get.test.js

```diff
@@ -26,14 +26,18 @@ describe("GET", () => {
         .expect(200)
         .then((result) => {
           expect(result.body.categories).toBeInstanceOf(Array);
-          result.body.categories.forEach((category) => {
-            expect(category).toEqual(
-              expect.objectContaining({
-                slug: expect.any(String),
-                description: expect.any(String),
-              }),
-            );
-          });
+          if (result.body.categories.length === 0) {
+            fail("no results");
```

Comment:
I'd suggest this isn't a best practice way of working, could this be swapped for a jest matcher?